### ISSUE TITLE
Gitlab mirror is gone

### ISSUE THREAD

**Issue by 9034725985** [bernaferrari/ChangeDetection#22]
What happened to our gitlab mirror? 

It used to be at https://gitlab.com/changedetection/ChangeDetection/ but the group https://gitlab.com/changedetection doesn't have anything anymore

**Comment by bernaferrari** [comment]
Oh, I deleted it. I was having some issues with repo out of sync, and I saw it had no stars or anything else, I thought no one was looking at it, so ended up removing it. BUT, since you clearly noticed it, if you want, I can re-mirror it there.

**Comment by 9034725985** [comment]
I thought the point was to use it for automated testing since it is a public repo. We can set it up to mirror GitHub and silently accept changes although imo we shouldn't rewrite history unless there's no way around it.

**Comment by bernaferrari** [comment]
I tried, but wasn't able to force the changes. Yeah, let's say.. Sometimes accidentally pushing the keystore password isn't what you want - but after converting the build.gradle to kts, I made sure this will (hopefully) never happen again. As for the tests, I usually look at Circle CI and Travis, I even forgot the gitlab repo existed - but again, if that is your wish, I have no problem making it again.
